File: php/Product.php
<?php
class Product
{
    private $idProduct = -1;
    private $name = "";
    private $brand = "";
    private $price = -1;

    public function __construct($idProduct, $name, $brand, $price)
    {
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    // setter getter
    public function getId(): int
    {
        return $this->idProduct;
    }

    public function setId($id)
    {
        $this->idProduct = $id;
    }

    public function getName(): string
    {
        return $this->name;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function getBrand(): string
    {
        return $this->brand;
    }

    public function setBrand($brand)
    {
        $this->brand = $brand;
    }

    public function getPrice(): int
    {
        return $this->price;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }
}

File: python/Product.py
class Product:
    __idProduct = -1
    __name = ""
    __brand = ""
    __price = -1

    # contructor
    def __init__(self, id, name, brand, price):
        self.__idProduct = id
        self.__name = name
        self.__brand = brand
        self.__price = price

    # setter getter

    def getId(self):
        return self.__idProduct

    def setId(self, Id):
        self.__idProduct = Id

    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

    def getBrand(self):
        return self.__brand

    def setBrand(self, brand):
        self.__brand = brand

    def getPrice(self):
        return self.__price

    def setPrice(self, price):
        self.__price = price
